modules/network-observability-working-with-conversations.adoc

```diff
@@ -19,27 +19,26 @@ As an administrator, you can you can group network flows that are part of the sa
 +
-[id="network-observability-flowcollector-configuring-conversations_{context}"]
 .Configure `FlowCollector` for conversation tracking
 [source, yaml]
 ----
-apiVersion: flows.netobserv.io/v1alpha1
+apiVersion: flows.netobserv.io/v1beta2
 kind: FlowCollector
 metadata:
   name: cluster
 spec:
  processor:
-  conversationEndTimeout: 10s                  <1>
-  logTypes: FLOWS                              <2>
-  conversationHeartbeatInterval: 30s           <3>
+  logTypes: Flows                              <1>
+  advanced:
+   conversationEndTimeout: 10s                 <2>
+   conversationHeartbeatInterval: 30s          <3>
 ----
-<1> The *Conversation end* event represents the point when the `conversationEndTimeout` is reached or the TCP flag is intercepted.
-<2> When `logTypes` is set to `FLOWS`, only the *Flow* event is exported. If you set the value to `ALL`, both conversation and flow events are exported and visible in the *Network Traffic* page. To focus only on conversation events, you can specify `CONVERSATIONS` which exports the *Conversation start*, *Conversation tick* and *Conversation end* events; or `ENDED_CONVERSATIONS` exports only the *Conversation end* events. Storage requirements are highest for `ALL` and lowest for `ENDED_CONVERSATIONS`.
+<1> When `logTypes` is set to `Flows`, only the *Flow* event is exported. If you set the value to `All`, both conversation and flow events are exported and visible in the *Network Traffic* page. To focus only on conversation events, you can specify `Conversations` which exports the *Conversation start*, *Conversation tick* and *Conversation end* events; or `EndedConversations` exports only the *Conversation end* events. Storage requirements are highest for `All` and lowest for `EndedConversations`.<1> The *Conversation end* event represents the point when the `conversationEndTimeout` is reached or the TCP flag is intercepted.
 <3> The *Conversation tick* event represents each specified interval defined in the `FlowCollector` `conversationHeartbeatInterval` parameter while the network connection is active.
 +
 [NOTE]
 ====
-If you update the `logType` option, the flows from the previous selection do not clear from the console plugin. For example, if you initially set `logType` to `CONVERSATIONS` for a span of time until 10 AM and then move to `ENDED_CONVERSATIONS`, the console plugin shows all conversation events before 10 AM and only ended conversations after 10 AM.
+If you update the `logType` option, the flows from the previous selection do not clear from the console plugin. For example, if you initially set `logType` to `Conversations` for a span of time until 10 AM and then move to `EndedConversations`, the console plugin shows all conversation events before 10 AM and only ended conversations after 10 AM.
```

Comment:
@jpinsonneau just checking, this note doesn't need to be updated right? Is `EndedCoversations` still a logType: 
![image](https://github.com/openshift/openshift-docs/assets/77501157/c8177831-8dcb-4017-944b-861a8f86eeb1)

Reply:
These didn't changed indeed:
https://github.com/netobserv/network-observability-operator/blob/release-1.5/apis/flowcollector/v1beta1/flowcollector_types.go#L377-L382

```
	// `logTypes` defines the desired record types to generate. Possible values are:<br>
	// - `FLOWS` (default) to export regular network flows<br>
	// - `CONVERSATIONS` to generate events for started conversations, ended conversations as well as periodic "tick" updates<br>
	// - `ENDED_CONVERSATIONS` to generate only ended conversations events<br>
	// - `ALL` to generate both network flows and all conversations events<br>
	// +kubebuilder:validation:Optional
	// +kubebuilder:validation:Enum:="FLOWS";"CONVERSATIONS";"ENDED_CONVERSATIONS";"ALL"
	// +kubebuilder:default:=FLOWS
	LogTypes *string `json:"logTypes,omitempty"`
```

Reply:
Cool, TY for confirming :)